ts/src/bitmex.ts

```diff
@@ -225,30 +225,33 @@ export default class bitmex extends Exchange {
                 'oldPrecision': false,
                 'networks': {
                     'BTC': 'btc',
-                    'ETH': 'eth',
-                    'BSC': 'bsc',
-                    'BNB': 'bsc',
```

Comment:
BSC & BNB were out of scope of our maintained unified list, so we are concentrating on `BEP20` (or at maximum still supporting 'BSC' temporarily, while BNB should be absolutely removed, as it is actually a common name for BEP2 chain, not BSC)